src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import './variables.scss';
+@import './header.scss';
+@import './card.scss';
+@import './stars.scss';
+@import './main.scss';
+@import './header.scss';
```

Comment:
The 'header.scss' file is imported twice. Remove the duplicate import to avoid redundant processing and potential conflicts in styles.